exercises/practice/hello-world/GreeterTest.bat

```diff
@@ -0,0 +1,23 @@
+@echo off
+REM ---------------------------------------------------
+REM testThatGreeterReturnsTheCorrectGreeting
+REM ---------------------------------------------------
```

Comment:
I approve of your choice of all caps here, even if it is perhaps not "best practice" (your words) and subjective.  It communicates well.

Reply:
Thanks you, I think my friends agree on this, 'rem' meaning comment should be capitalized.